src/coreclr/vm/comutilnative.h

```diff
@@ -134,6 +134,7 @@ class GCInterface {
     static FORCEINLINE UINT64 InterlockedAdd(UINT64 *pAugend, UINT64 addend);
     static FORCEINLINE UINT64 InterlockedSub(UINT64 *pMinuend, UINT64 subtrahend);
 
+    static FCDECL0(INT64,   _GetTotalPauseDuration);
     static FCDECL2(void,    GetMemoryInfo, Object* objUNSAFE, int kind);
```

Comment:
none of the existing methods start with '_', is there a reason why this one does?